chain/client-primitives/src/types.rs

```diff
@@ -133,8 +133,8 @@ impl ToString for ShardSyncStatus {
             ShardSyncStatus::StateDownloadScheduling => "scheduling".to_string(),
             ShardSyncStatus::StateDownloadApplying => "applying".to_string(),
             ShardSyncStatus::StateDownloadComplete => "download complete".to_string(),
-            ShardSyncStatus::StateSplitScheduling => "split scheduling".to_string(),
-            ShardSyncStatus::StateSplitApplying => "split applying".to_string(),
+            ShardSyncStatus::ReshardingScheduling => "split scheduling".to_string(),
```

Comment:
Is it safe to change the ToString value here? Say from "split scheduling" to "resharding scheduling"? I don't think we store this anywhere, but potentially might need to revisit logging/dashboards.

Reply:
Good catch, let's do this. I don't know of any dependencie. Either way we should be fine as long as nobody upgrades to this version in the middle of resharding which shouldn't happen anyway.